worker.go

```diff
@@ -266,25 +275,33 @@ func buildManagedModel(certs []*certs.CertificateSummary, certsPerALB int, certT
 		// try to add ingress to existing ALB stacks until certificate
 		// limit is exeeded.
 		added := false
-		for _, lb := range model {
+		for _, lb := range loadBalancers {
 			if lb.AddIngress(certificateARNs, ingress, certsPerALB) {
 				added = true
 				break
 			}
 		}
 
 		// if the ingress was not added to the ALB stack because of
-		// non-matching scheme or too many certificates, add a new
+		// non-matching scheme, non-matching security group or too many certificates, add a new
 		// stack.
 		if !added {
 			i := make(map[string][]*kubernetes.Ingress, len(certificateARNs))
 			for _, certificateARN := range certificateARNs {
 				i[certificateARN] = []*kubernetes.Ingress{ingress}
 			}
-			model = append(model, &loadBalancer{ingresses: i, scheme: ingress.Scheme, shared: ingress.Shared})
+			loadBalancers = append(loadBalancers, &loadBalancer{ingresses: i, scheme: ingress.Scheme, shared: ingress.Shared, securityGroup: ingress.SecurityGroup})
 		}
 	}
 
+	return loadBalancers
+}
+
+func buildManagedModel(certs []*certs.CertificateSummary, certsPerALB int, certTTL time.Duration, ingresses []*kubernetes.Ingress, stacks []*aws.Stack) []*loadBalancer {
+	sortStacks(stacks)
+	model := getAllLoadBalancers(certTTL, stacks)
+	model = matchIngressesToLoadBalancers(model, certs, certsPerALB, ingresses)
```

Comment:
You don't need really to assign the value `model` if you change the var in matchIngressesToLoadBalancers that you pass, because a slice is reference value and we have pointers as values inside. 
If you think it's more readable like this, then it's fine to just let it like this.
Comment is more FYI ;)

Reply:
Thanks ;) In my opinion it's just more readable and looks exactly how `append` look like and what actually is done here is really appending to the slice under some conditions.